inc/class-wp-cli-command.php

```diff
@@ -99,6 +99,39 @@ public function save( $args, $args_assoc ) {
 			if ( ! empty( $args_assoc['verbose'] ) ) {
 				WP_CLI::line( 'Saving ' . $url );
 			}
+
+			$options = [
+				'user_id' => get_current_user_id(),
```

Comment:
Remark: Keep in mind that by default, there is no current user in the WP CLI context, see https://danielbachhuber.com/tip/run-wp-cli-command-wordpress-user/

Reply:
Yep. Noted. `user_id` here would be optional